features/cells/src/main/java/com/wire/android/feature/cells/ui/createfolder/CreateFolderScreen.kt

```diff
@@ -0,0 +1,127 @@
+import com.wire.android.ui.common.textfield.WireTextField
+import com.wire.android.ui.common.topappbar.NavigationIconType
+import com.wire.android.ui.common.topappbar.WireCenterAlignedTopAppBar
+import com.wire.android.ui.theme.WireTheme
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import java.util.Locale
+
+@Destination(
+    style = PopUpNavigationAnimation::class
+)
+@Composable
+fun CreateFolderScreen(
+    navigator: WireNavigator,
+    modifier: Modifier = Modifier,
+) {
+    WireScaffold(
+        modifier = modifier,
+        topBar = {
+            WireCenterAlignedTopAppBar(
+                onNavigationPressed = { navigator.navigateBack() },
+                navigationIconType = NavigationIconType.Close(),
+                elevation = dimensions().spacing0x,
+                title = stringResource(id = R.string.cells_create_folder),
+            )
+        },
+        bottomBar = {
+            AnimatedVisibility(
+                visible = true,
+                enter = fadeIn() + expandVertically(),
+                exit = shrinkVertically() + fadeOut(),
+            ) {
+                Surface(
+                    color = MaterialTheme.wireColorScheme.background,
+                    shadowElevation = MaterialTheme.wireDimensions.bottomNavigationShadowElevation
+                ) {
+                    Column(
+                        verticalArrangement = Arrangement.Center,
+                        modifier = Modifier
+                            .padding(dimensions().spacing16x)
+                    ) {
+                        WirePrimaryButton(
+                            text = stringResource(R.string.cells_create_folder),
+                            onClick = {},
+                            state = WireButtonState.Default,
+                            clickBlockParams = ClickBlockParams(blockWhenSyncing = true, blockWhenConnecting = true),
+                        )
+                    }
+                }
+            }
+        }
+    ) {
+        WireTextField(
+            textState = TextFieldState(),
+            placeholderText = stringResource(R.string.cells_folder_name),
+            labelText = stringResource(R.string.cells_folder_name).uppercase(Locale.getDefault()),
+            modifier = Modifier
+                .padding(it)
+                .padding(
+                    top = dimensions().spacing32x,
+                    start = dimensions().spacing16x,
+                    end = dimensions().spacing16x
+                )
+        )
+    }
+}
+
+@MultipleThemePreviews
+@Composable
+fun PreviewCreateFolderScreen() {
+    val dummyNavigator = object : WireNavigator {
+        override fun navigate(navigationCommand: NavigationCommand, onlyIfResumed: Boolean) {
+            TODO("Not yet implemented")
+        }
+
+        override fun navigateBack(onlyIfResumed: Boolean) {
+            TODO("Not yet implemented")
+        }
+    }
```

Comment:
I would suggest to create such object in the `WireNavigator.kt`, something like:
```
object PreviewNavigator : WireNavigator {
    override fun navigate(navigationCommand: NavigationCommand, onlyIfResumed: Boolean) { /* No-op */ }
    override fun navigateBack(onlyIfResumed: Boolean) { /* No-op */ }
}
```
so that we could reuse it everywhere 💪